### ISSUE TITLE
Possible bugs in column utility

### ISSUE THREAD

**Issue by dliuzzi** [Alexpux/MSYS2-packages#680]
## TL;DR

 - What commands did you run to trigger this issue?

    <details>
      <summary>Click to expand</summary>

      **testcol.sh**

      ```sh
      c0='\033[0m'
      c1='\033[31m'
      c2='\033[32m'
      c3='\033[34m'

      text="\
      1 \
      lorem \
      ipsum \
      ${c1}dolor${c0} \
      ${c2}sit${c0} \
      ${c3}amet${c0} \

      2 \
      consectetur \
      adipisicing \
      ${c1}elit${c0} \
      ${c2}sed${c0} \
      ${c3}do${c0} \

      3 \
      eiusmod \
      tempor \
      ${c1}incididunt${c0} \
      ${c2}ut${c0} \
      ${c3}labore${c0} \

      4 \
      et \
      dolore \
      ${c1}magna${c0} \
      ${c2}aliqua${c0} \
      ${c3}ut${c0} \

      5 \
      enim \
      ad \
      ${c1}minim${c0} \
      ${c2}veniam${c0} \
      ${c3}quis${c0} \

      6 \
      nostrud \
      exercitation \
      ${c1}ullamco${c0} \
      ${c2}laboris${c0} \
      ${c3}nisi${c0} \

      7 \
      ut \
      aliquip \
      ${c1}ex${c0} \
      ${c2}ea${c0} \
      ${c3}commodo${c0} \

      8 \
      consequat \
      duis \
      ${c1}aute${c0} \
      ${c2}irure${c0} \
      ${c3}dolor${c0} \

      9 \
      in \
      reprehenderit \
      ${c1}in${c0} \
      ${c2}voluptate${c0} \
      ${c3}velit${c0} \

      10 \
      esse \
      cillum \
      ${c1}dolore${c0} \
      ${c2}eu${c0} \
      ${c3}fugiat${c0} \
      "

      printf "$text" | column -t
      ```
    </details>

 - What did you expect to occur after running these commands?

 _Columns should line up properly; "line too long" error (row 10 missing EOL)_

    ![ubuntu](https://cloud.githubusercontent.com/assets/143884/18093298/b5ee61ba-6ecf-11e6-94dd-1e0bc01a2a69.png)
— Bash on Ubuntu on Windows

 - What actually happened instead?

 _Columnn alignment lost after ANSI escape chars; row 10 displayed_

    ![windows](https://cloud.githubusercontent.com/assets/143884/18093340/d468950c-6ecf-11e6-8db8-faf66d4cc841.png)
— MSYS2

------

I came across this while using Paul Irish's excellent [git-recent](https://github.com/paulirish/git-recent). This tool shows you a nicely formatted list of your latest local git branches. Internally, it utility uses [`column -t`](https://github.com/paulirish/git-recent/blob/b0cca7800b4855eb93ffda5cf2bc946ce9497f67/git-recent#L21) and [works beautifully](https://cloud.githubusercontent.com/assets/39191/17446638/039d4cee-5aff-11e6-9e11-4294f0020513.png) on macOS and Ubuntu. Windows, on the other hand, completely [butchers the table layout](https://cloud.githubusercontent.com/assets/143884/17586600/014d0cb8-5fc3-11e6-884e-a5ba503df522.png).

Although I've [reported this on git-recent's issue tracker](https://github.com/paulirish/git-recent/issues/8), this actually seems to have more to do with column than git-recent itself, as I discovered the problem disappears if I temporarily remove the text coloring.

Being utterly ignorant about MSYS, I went on to report this on [Git for Windows's issue tracker](Git for Windows) as well, since that's what installed `column` in my system. The gentlemen there kindly refereed me to here, as `column` actually comes from MSYS2, and Git for Windows merely bundles it. So here I am. I have included the relevant bits of that issue here.

**NOTE: I did my best to try and identify the proper place to report this, but I'm still not 100% sure, so apologies in advance if this is the right one.**
